src/services/codefixes/addMissingConst.ts

```diff
@@ -104,6 +104,8 @@ namespace ts.codefix {
             return every([expression.left, expression.right], expression => expressionCouldBeVariableDeclaration(expression, checker));
         }
 
-        return isIdentifier(expression.left) && !checker.getSymbolAtLocation(expression.left);
+        return expression.operatorToken.kind === SyntaxKind.EqualsToken
+            && isIdentifier(expression.left)
+            && !checker.getSymbolAtLocation(expression.left);

```

Comment:
Semi-unrelated, but converting `verify.codeFixAll` to `verify.codeFix` in a fourslash test that contained `yield* this.y()` in a non-generator function revealed that we were offering to add `const` before _any_ binary expression with unresolved names, not just binary expressions with `=` operators.